README.md

```diff
@@ -1,12 +1,12 @@
 # Notes
 
-[![build state](https://travis-ci.org/owncloud/notes.png)](https://travis-ci.org/owncloud/notes)[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/owncloud/notes/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/owncloud/notes/?branch=master)
+[![build state](https://travis-ci.org/owncloud/notes.png)](https://travis-ci.org/nextcloud/notes)[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/nextcloud/notes/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/nextcloud/notes/?branch=master)
 
-The Notes app is a distraction free notes taking app. It offers a [RESTful API](https://github.com/owncloud/notes/wiki/API-0.2) for app developers. The source code is [available on GitHub](https://github.com/owncloud/notes)
+The Notes app is a distraction free notes taking app. It offers a [RESTful API](https://github.com/nextcloud/notes/wiki/API-0.2) for app developers. The source code is [available on GitHub](https://github.com/nextcloud/notes)
 
 ![Screenshot of ownCloud Notes](https://cloud.githubusercontent.com/assets/4741199/17731273/e557b5fe-646c-11e6-9975-d6b242454482.png)
```

Comment:
The screenshot was taken by @stefan-niedermann, maybe he has still the original (plain) text (and list of notes) and could create a new one.